### DISCUSSION TITLE
Lua 5.4

### DISCUSSION THREAD

**Discussion by yurimaturana** [zeusssz/hercules-obfuscator#34] [Q&A] [answered]
Have suport with lua 5.4?

goto ::
++ and --
< const >

**Comment by zeusssz** [comment, marked as answer]
lua 5.4 is the supported version currently

**Comment by yoursvexyyy** [comment]
what is jayobfuscator

**Reply by zeusssz** [reply]
no idea, but i presume youre referring to this
![image](https://github.com/user-attachments/assets/2714faf5-2975-4d98-9edb-3ddf5448f6f3)

**Reply by yoursvexyyy** [reply]
Yes I'm finding since a long time but I couldn't
